python/python/glide/async_commands/core.py

```diff
@@ -3157,6 +3157,71 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def zinter(
+        self,
+        keys: List[str],
+    ) -> List[str]:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and returns a list of intersecting elements.
+        To get the scores as well, see `zinter_withscores`.
+        To store the result in a key as a sorted set, see `zinterstore`.
+
+        See https://valkey.io/commands/zinter/ for more details.
```

Comment:
Please add a cross slot notice to all functions

Reply:
Addressed